torch/_export/verifier.py

```diff
@@ -307,7 +308,7 @@ def _verify_exported_program_signature(exported_program) -> None:
         )
 
     for input_spec, node in zip(gs.input_specs, input_node_names):
-        if isinstance(input_spec.arg, (TensorArgument, SymIntArgument)):
+        if isinstance(input_spec.arg, (TensorArgument, SymIntArgument, SymBoolArgument)):
```

Comment:
When we have SymBoolArgument inputs? Can add a test for it?

Reply:
@ydwu4 I am not sure, I was just following the pattern of SymIntArgument.

Any SymIntArgument test that I can take a look?